website/docusaurus.config.js

```diff
@@ -415,6 +415,22 @@ module.exports = {
       copyright:
         "Copyright © 2019-2021 The Apache Software Foundation. Apache APISIX, APISIX™, Apache, the Apache feather logo, and the Apache APISIX project logo are either registered trademarks or trademarks of the Apache Software Foundation.",
     },
+    announcementBar: {
+      id: 'apisix-2.6-Release',
+      backgroundColor: "#e8433e",
+      textColor: 'white',
+      content:
+        '➡️ Apache APISIX v2.6.0 released, users can view Changelogs <a target="_blank" rel="noopener noreferrer" href="https://github.com/apache/apisix/blob/2.6/CHANGELOG.md#260">APISIX 2.6.0 release</a>! 🔄',
+    },
+    /*
+    announcementBar: {
+      id: 'query',
```

Comment:
Do we need to keep this?

Reply:
Yes that will make easier for us to change the announcement just by un commenting it, docusaurus and many websites uses the same strategy